### ISSUE TITLE
How to run kanel on a forked version of this repo?

### ISSUE THREAD

**Issue by christiaanwesterbeek** [kristiandupont/kanel#370]
I have made a [fork](https://github.com/christiaanwesterbeek/kanel/tree/additional-views-support) from this repo with some minor changes that will also try to look for a target in the views. 

The problem is that when I run `yarn exec kanel` that it runs the version of https://github.com/kristiandupont/kanel.

Let's dive in.

I've added my fork to some repo like this.

`yarn add -D https://github.com/christiaanwesterbeek/kanel#additional-views-support`

In the dependencies in node_modules, I can actually see my changes here
<img width="879" alt="image" src="https://user-images.githubusercontent.com/465989/217864615-1b1d768d-331f-4689-aaee-32960b5d8521.png">

But not here
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/465989/217864787-3a039635-8612-4d76-b99c-bb883c6e7f32.png">

So, my question is how can I add my fork of kanel to some repo and execute kanel using my forks code, instead of kristiandupont's?

**Comment by kristiandupont** [comment]
I might be missing something but it looks like your version wasn't built? Did you run `yarn build` inside `/packages/kanel`?

**Comment by christiaanwesterbeek** [comment]
When in my local clone of my fork of kanel, I run `yarn build` inside /packages/kanel, I indeed see my changes in /packages/kanel/src/generators/resolveType.ts are built in /packages/kanel/build/generators/resolveType.js. The thing is that that directory is ignored, so not committed.

Now back to my application, where I have my fork as a dependency. This is the package.json of that application

```
...
"devDependencies": {
    "dotenv": "^16.0.3",
    "eslint-plugin-ignore-generated": "^3.1.0",
    "kanel-monorepo": "https://github.com/christiaanwesterbeek/kanel#additional-views-support"
}
```

The 2 screenshots in my first comment are from that application. As you can see, in the build directory there is not what would be the build result of my code

**Comment by kristiandupont** [comment]
Looking at your screenshots, you see your changes in a `src` folder that sits right underneath `bin`. In the second screenshot, the folder right underneath `bin` is the `build` folder. Are you referring to the same `node_modules` folder?

**Comment by kristiandupont** [comment]
Right, I think I understood that the second screenshot is from your clone, and the first one is from where you installed it?

**Comment by kristiandupont** [comment]
What does
```bash
ls -la node_modules/.bin/kanel
```
give you?
